lib/matplotlib/colors.py

```diff
@@ -1213,8 +1213,8 @@ class SymLogNorm(Normalize):
     *linthresh* allows the user to specify the size of this range
     (-*linthresh*, *linthresh*).
     """
-    def __init__(self, linthresh, linscale=1.0,
-                 vmin=None, vmax=None, clip=False):
+    def __init__(self, linthresh, linscale=1.0, vmin=None, vmax=None,
+                 clip=False, base=None):
```

Comment:
```suggestion
                 clip=False, *, base=None):
```
For now, only the new parameter `base` can be kw-only. Making other parameters kw-only needs a deprecation first.